bodhi/server/services/updates.py

```diff
@@ -189,6 +194,36 @@ def set_request(request):
     return dict(update=update)
 
 
+@update_rerun_test.post(schema=bodhi.server.schemas.RerunTestSchema,
```

Comment:
I think ```flake8``` might get upset about the ```schema``` not vertically aligning with the other keys. I suggest moving it to the next line to appease the PEP-8 gods.